.github/workflows/lint-pr-title.yml

```diff
@@ -0,0 +1,59 @@
+name: "Lint PR title"
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+      - labeled
+      - unlabeled
+
+jobs:
+  main:
+    name: Validate PR title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-semantic-pull-request@v5
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          # Configure which types are allowed (newline-delimited).
+          # Default: https://github.com/commitizen/conventional-commit-types
+          #types:
+
+          # Configure which scopes are allowed (newline-delimited).
+          # These are regex patterns auto-wrapped in `^ $`.
+          scopes: |
+            \d+
+
+          # Configure that a scope must always be provided.
+          requireScope: true
+
+          # Configure which scopes are disallowed in PR titles (newline-delimited).
+          # For instance by setting the value below, `chore(release): ...` (lowercase)
+          # and `ci(e2e,release): ...` (unknown scope) will be rejected.
```

Comment:
I am not sure what this comment it trying to say

all of these `chore(release): ...` and `ci(e2e,release): ...` will be rejected or just `(unknown scope)`?

based on the regex all of these are allowed because do not match `release` and no uppercase letter

Reply:
Yeah the comment is a bit confusing. I'd just copied it from the readme.
I think it means that any message with a scope that is either "release" or has upper-case letters would be disallowed.